site/content/docs/0.2/extend/icons.md

```diff
@@ -1,11 +1,733 @@
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#vector"/>
+  </svg>
+  <h1 class="mb-none">H1 with large icon and a very long title to see how it behaves while being wrapped</h1>
+</div>
+{{< /example >}}
+
+{{< example >}}
+<div>
+  <svg class="bm-lg-icon mb-scaled-shortest text-status-info" width="1rem" height="1rem" fill="currentColor" aria-hidden="true">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/ouds-web-sprite.svg#vector"/>
+  </svg>
+  <p class="mb-none">
+    Default long paragraph with large icon. It stands here in order for you to see how the icon should behave when it's placed next to a long paragraph. Whenever the paragraph stands on multiple lines please prefer using a block element and not an inline element so there will no strange behaviors.
+  </p>
+</div>
+{{< /example >}}
+
+### `<img>` and font icon
+
+SVG Sprite is the preferred choice for flexibility, performance and accessibility, and this is why all our examples use this method but there are other ways to display icons. They should all behave the same thanks to our CSS. If you are experimenting any trouble, feel free to search or fill in a new [Github discussion](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/discussions) or contact [our team]({{< docsref "/about/team" >}}). <!-- Check out for more information in [our documentation]({{< docsref "/extend/icons" >}}). -->
+
+{{< example >}}
+<div class="d-flex align-items-center mb-medium">
+  <img class="hxl-lg-icon me-short" alt="" src="/docs/{{< param docs_version >}}/assets/img/vite.svg">
+  <h1 class="mb-none">H1 with large icon</h1>
+</div>
+<div class="d-flex align-items-center mb-medium">
+  <span class="hxl-lg-icon me-short si si-settings" aria-hidden="true"></span>
+  <h1 class="mb-none">H1 with large icon</h1>
+</div>
+{{< /example >}}
+
+## Icons accessibility
+
+{{< callout info >}}
+For more details, **Orange Accessibility Guidelines** provides [a deep-dive article regarding SVG accessibility](https://a11y-guidelines.orange.com/en/articles/accessible-svg/).
+{{< /callout >}}
+
+#### Decorative icons
+Purely **decorative icons** (like repeating information of an adjacent text) must be hidden to assistive technologies:
+- for `<svg>` or `<span>` tags, use the attribute `aria-hidden="true"`
+- for `<img>`, use an empty `alt` attribute
+- CSS background images are intended to be decorative
+
+#### Informative/meaningful icons
+If the icon is **meaningful**, e.g. only content of a button, you have to provide an appropriate alternative text: for example, the description of the icon or the description of the action triggered. The best way to do this is to keep the icon hidden to assistive technologies (see above) and add a visually hidden label (which will be perceived by assistive technologies) by using the `.visually-hidden` class. For external images, you can also fill the `alt` attribute directly.
```

Comment:
My adding for these last sentences: 
The best way to do this is to keep the icon hidden to assistive technologies (see above) and add a visually hidden label (which will be perceived by assistive technologies) by using the `.visually-hidden` class. For external images, you can also fill in the `alt` attribute directly. Instead of these two techniques, you can also use an `aria-label` in the `button` tag to provide an accessible name.

Reply:
done